src/app/static/src/app/models/project.ts

```diff
@@ -66,12 +66,4 @@ export class Project {
         this.regions = regionsName.map(n => new Region(n, this, baselineOptions, interventionOptions));
         this.currentRegion = currentRegion || this.regions[0]
     }
-
-    setCurrentRegion(url: string) {
```

Comment:
Since the state needs to be serialisable, these classes can't have methods. This logic has been moved to the `SetCurrentRegion` mutation.